seed/landing/views.py

```diff
@@ -52,41 +49,36 @@ def login_view(request):
                 username=form.cleaned_data['email'].lower(),
                 password=form.cleaned_data['password']
             )
-            if new_user and new_user.is_active:
+            if new_user is not None and new_user.is_active:
```

Comment:
I'm curious why you had to add the `is not None` clause here.  Seems like this is equivalent to before, but more wordy...hmm.

Reply:
i am just more verbose i guess. For some reason I thought there was a difference between `if new_user` and `if user is not None`